i18n/en.json

```diff
@@ -2648,6 +2652,9 @@
   "posts_view.maxLoaded": "Looking for a specific message? Try searching for it",
   "posts_view.newMsg": "New Messages",
   "posts_view.newMsgBelow": "New {count, plural, one {message} other {messages}}",
+  "promote_to_user_modal.desc": "This action promotes the guest {username} to be a regular user. The user will be able to contact with users outside of his channels and join to public channels by themselves. Are you sure you want to promote guest {username} to user?",
```

Comment:
Recommend:  "This action promotes the guest {username} to a member. This action will allow the user to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to promote guest {username} to user?"

Reply:
Done :)